### PR TITLE
feat: edit sql chart from dashboard

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #11148

### Description:

I didn't add the tooltip on disabled edit mode, to make the code more readable. Let me know if you think it is needed. 

[Screencast from 29-08-24 12:56:18.webm](https://github.com/user-attachments/assets/2b11c0f4-9059-4048-9306-ca8f31f2e7c5)

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx [hunk 1]
```diff
@@ -27,7 +31,33 @@ interface Props
  * Handle minimal mode
  * handle tabs
  */
-
+const DashboardOptions = ({
```
Comment (joaoviana) [inline packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx:34]: 💪

File: packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx [hunk 2]
```diff
@@ -27,7 +31,33 @@ interface Props
...
+}: {
+    isEditMode: boolean;
+    data: {
+        resultsAndColumns: ResultsAndColumns;
+        chart: SqlChart;
+    };
+}) => {
+    const history = useHistory();
+    return (
+        <Box>
```
Comment (joaoviana) [inline packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx:46]: do we need `Box`?

**Review by joaoviana — APPROVED** [review]
tested in Render!

File: packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx (hunk 2 above)
Comment (rephus) [reply packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx:46]: that's what the other options on charts were using, so I added it here too

**Pull request merged**